samples/samples-js-v4/host.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": "2.0",
+  "logging": {
```

Comment:
Copy the host.json from the other samples so they're all aligned (should have default log level of trace)